recipe/meta.yaml

```diff
@@ -115,6 +105,33 @@ outputs:
         - LICENSE
         - napari/_vendor/darkdetect/LICENSE
         - napari/_vendor/qt_json_builder/LICENSE
+      summary: a fast n-dimensional image viewer in Python, with only the required dependencies
+      doc_url: http://napari.org
+      dev_url: https://github.com/napari/napari
+
+  - name: napari
+    version: {{ version }}
+    build:
+      noarch: python
+    requirements:
+      run:
+        - {{ pin_subpackage('napari-base', exact=True) }}
+        - fsspec
+        - napari-plugin-manager >=0.1.0a2,<0.3.0a0
+        - napari-svg >=0.1.8
+        - numba >=0.57.1
+        - pooch >=1.3.0
+        # - py-triangle  # a bit fragile with bad data, leave out for now
+        - zarr >=2.12.0
+    test:  # just to check that the environment can indeed solve
+      requires:
+        - pyside2  # use this opportunity to test solvability with other backend
```

Comment:
We tested installability with pyqt in the output above (`napari-base`). In here we just pick `pyside2` because we can now, given the new output. We are _not_ running the test suite here, just installability checks and minimum "this thing runs", import wise.

Reply:
Great, thanks!